Lesson5/src/Car.java

```diff
@@ -0,0 +1,69 @@
+public class Car extends groundTransport {
+    private int capacity;
+    private CarBody carBody;
+
+    public Car(int power, int maxSpeed, int weight, String brand, int countOfWheels, float fuelConsumption, CarBody carBody) {
+        super(power, maxSpeed, weight, brand, countOfWheels, fuelConsumption);
+        this.carBody = carBody;
+        setCapacity();
+    }
+
+    public int getCapacity() {
+        return capacity;
+    }
+
+    public void setCapacity() {
+        switch (carBody) {
+            case SEDAN:
+                this.capacity = CarBody.SEDAN.getCapacitty();
+                break;
+            case WAGON:
+                this.capacity = CarBody.WAGON.getCapacitty();
+                break;
+            case PICKUP:
+                this.capacity = CarBody.PICKUP.getCapacitty();
+                break;
+            case CABRIOLET:
+                this.capacity = CarBody.CABRIOLET.getCapacitty();
+                break;
+            case COUPE:
+                this.capacity = CarBody.COUPE.getCapacitty();
+                break;
+        }
+
+    }
+
+    public CarBody getBody() {
+        return carBody;
+    }
+
+    public void setBody(CarBody carBody) {
+        this.carBody = carBody;
+    }
+
+    public void getMaxDistance(float time) {
+        float maxDistance = getMaxSpeed() * time;
+        float fuelSpend = (maxDistance * getFuelConsumption()) / 100;
```

Comment:
homework said "расчёт израсходуемого топлива вынести в отдельный метод private." =)
so please make some refactoring here. It makes sense to me because the calculation of spent fuel can change in the future, but it shouldn't affect code of this method